jose/jwt.py

```diff
@@ -455,7 +459,7 @@ def _validate_at_hash(claims, access_token, algorithm):
         raise JWTClaimsError("at_hash claim does not match access_token.")
 
 
-def _validate_claims(claims, audience=None, issuer=None, subject=None, algorithm=None, access_token=None, options=None):
+def _validate_claims(now, claims, audience=None, issuer=None, subject=None, algorithm=None, access_token=None, options=None):
```

Comment:
Why change the function signature? Wouldn't it be easier to append `now=None` to the end of the parameter list and keep the function signature the same? And then use

```python
now = now or datetime.utcnow()
```

to default to `datetime.utcnow()`, just in case anybody is using these private methods.

I realize that they're private methods, and people are signing up to keep their code up-to-date when they use these private methods, but there's no point in code churn without a good reason.

Reply:
It was not part of the public API (this particular function does not even have docstring to serve as a documentation), so there was no hesitation to change the signature. Making `now` (required) positional argument better states the intention to me.

Either way, updated.